core/kernel/user_ta.c

```diff
@@ -49,12 +49,20 @@
 #include <utee_defines.h>
 #include <util.h>
 
-static void init_utee_param(struct utee_params *up,
-			const struct tee_ta_param *p, void *va[TEE_NUM_PARAMS])
+static TEE_Result init_utee_param(struct utee_params *up,
+				  const struct tee_ta_param *p,
+				  void *va[TEE_NUM_PARAMS])
 {
+	TEE_Result res = TEE_SUCCESS;
 	size_t n;
```

Comment:
nitpciking: by the way, could you initialize `n` to 0, even if it's a bit offtopic in this change?

Reply:
Sure. updated